about.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+	<link rel="stylesheet" href="style.css">
+	<script defer src="script.js"></script>
+  <title>About Us</title>
+</head>
+
+<body>
+  <!-- Header & Navigation -->
+  <header>
+	<!-- header-1 -->
+    <div class="first-header">
+      <ul class="first-header-ul">
+        <li><a><img src="Img/facebook-letter-logo-svgrepo-com.svg" alt="Facebook"></a></li>
+        <li><a><img src="Img/twitter-white.svg" alt="twitter"></a></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+	<!-- header-2 -->
+    </div>
+    <div class="nav-div">
+      <div class="logo">
+					<a href="index.html"
+						><img class=logo src="Img/daankwaLogo.png"  alt="Logo"
+					/></a>
+				</div>
+				<ul class="nav justify-content-end">
+					<li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#program">Program</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="./index.html#join">Join</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="./index.html#sponsors">Sponsor</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">News</a>
+        </li>
+        <li class="nav-item campaign">
+          <a class="nav-link" href="#">Talents</a>
+        </li>
+      </ul>
+      <div class="hamburger">
```

Comment:
- [ ] While making the changes suggested by the previous reviewer, you don't have the background color for the `Talents` nav link and this makes it not easier to see so make sure the `Talents` like have the same background color as the other mobile menu nav links. :pray: You can see this below

  ![Screenshot from 2023-06-30 00-59-31](https://github.com/obibaadoma/captone_DaaNkwa/assets/92303891/dd0812fd-911a-47e1-8f63-cc55880f1459)